src/display.js

```diff
@@ -61,4 +61,55 @@ Display.prototype.completeTodo = function (e) {
   item.classList.toggle('completed');
 };
 
+Display.prototype.editTodo = function (e) {
+  const item = e.target.parentElement.parentElement;
+  const id = item.dataset.createdate;
+  const data = ls.findItem(id);
+
+  document.getElementById('newTaskID').value = data.title;
+  document.getElementById('updateTaskId').value = data.id;
+
+  document.querySelector('.AddTaskBtn').style.display = 'none';
+  document.querySelector('.EditTaskBtn').style.display = 'inline';
+  document.querySelector('.CancelTaskBtn').style.display = 'inline';
+};
+
+Display.prototype.updateTodo = function () {
+  const todoId = document.getElementById('updateTaskId').value;
+  const todoTile = document.getElementById('newTaskID').value;
+  const items = document.querySelectorAll('.taskTitle');
+
+  if (todoTile.length > 0) {
+    ls.updateTodo(todoId, todoTile);
+    items.forEach((title) => {
+      if (title.parentElement.parentElement.dataset.createdate === todoId) {
+        title.innerText = todoTile;
+      }
+    });
+  }
+
+  document.getElementById('newTaskID').value = '';
+  document.getElementById('updateTaskId').value = '';
+
+  document.querySelector('.AddTaskBtn').style.display = 'inline';
+  document.querySelector('.EditTaskBtn').style.display = 'none';
+  document.querySelector('.CancelTaskBtn').style.display = 'none';
+};
+
+Display.prototype.cancelTodo = function () {
+  document.getElementById('newTaskID').value = '';
+  document.getElementById('updateTaskId').value = '';
+
+  document.querySelector('.AddTaskBtn').style.display = 'inline';
+  document.querySelector('.EditTaskBtn').style.display = 'none';
+  document.querySelector('.CancelTaskBtn').style.display = 'none';
+};
+
+Display.prototype.clearCompleted = function () {
```

Comment:
- [ ] wonderful job. App works and interact as expected. Please make your work more organize, kindly create a new js file imported as a module for the `clear all completed` methods and methods related to the status updates (completed: true / false).

Reply:
I have already implemented a module in the project. The initial assumption was that I would write it in one file for incorporation with Webpack. However, since I have already implemented modules, you can not ask about what is already in place.